keel-web/src/test/kotlin/com/netflix/spinnaker/keel/ApplicationServiceTests.kt

```diff
@@ -264,7 +264,8 @@ class ApplicationServiceTests : JUnit5Minutests {
               vetoed = listOf(),
               deploying = null,
               skipped = listOf()
-            )
+            ),
+            null
```

Comment:
Consider adding the property name here to make it clear what we're assigning `null` to.